packages/shared/lib/core/wallet/actions/hideActivity.ts

```diff
@@ -2,16 +2,31 @@ import { selectedAccount } from '@core/account'
 import { get } from 'svelte/store'
 import { updateActivity } from '../stores'
 import { hiddenActivities } from '../stores/hidden-activities.store'
+import { localize } from '@core/i18n'
+import { showAppNotification } from '@lib/notifications'
 
 export function hideActivity(id: string): void {
-    const accountId = get(selectedAccount).id
-    hiddenActivities.update((state) => {
-        if (!state[accountId] || !Array.isArray(state[accountId])) {
-            state[accountId] = []
-        }
-        state[accountId].push(id)
-        return state
-    })
+    try {
+        const accountId = get(selectedAccount).id
+        hiddenActivities.update((state) => {
+            if (!state[accountId] || !Array.isArray(state[accountId])) {
+                state[accountId] = []
+            }
+            state[accountId].push(id)
+            return state
+        })
 
-    updateActivity(accountId, { id, isHidden: true })
+        updateActivity(accountId, { id, isHidden: true })
+
+        showAppNotification({
+            type: 'info',
+            message: localize('general.rejectSuccess'),
+        })
+    } catch (err) {
+        console.error(err)
+        showAppNotification({
+            type: 'error',
+            message: localize('general.rejectFailed'),
```

Comment:
this is different in the en.json file
```suggestion
            message: localize('general.rejectError'),
```